server/src/main/java/org/elasticsearch/transport/OutboundMessage.java

```diff
@@ -163,6 +173,11 @@ public String toString() {
         }
     }
 
+    private static CompressionScheme adjustedScheme(Version version, CompressionScheme compressionScheme) {
+        // TODO: Change after backport
+        return compressionScheme == CompressionScheme.LZ4 && version.before(Version.V_8_0_0) ? null : compressionScheme;
```

Comment:
Can we centralize this version constant as `LZ4_VERSION` such that there is only one constant to change in the backport?